src/redux/actions/articles.js

```diff
@@ -72,3 +94,43 @@ export const createArticle = articleData => async (dispatch) => {
     return error.response.data;
   }
 };
+
+export const likeCurrentArticle = slug => async (dispatch) => {
```

Comment:
@youngestdj `likeCurrentArticle`? is there a place where we can be liking another type of article? If yes then this can stay, if no, then `likeArticle` should be enough.

Reply:
@9jaswag There's already a function named `likeArticle` in that file. I'll look for another suitable name.